python/lsst/ts/m2/tcp_client.py

```diff
@@ -363,12 +367,10 @@ async def close(self):
 
         self.log.info("Close the connection.")
 
-        # Write the EOF and close
-        if (self.writer is not None) and self.writer.can_write_eof():
-            self.writer.write_eof()
-
```

Comment:
You need to write the EOF to let the M2 server (EUI in LabVIEW) knows the connection is closed. Otherwise, the EUI will think the connection is still there even though the client closes the connection.

Reply:
ah!  see.. that makes sense..